.tool-versions

```diff
@@ -0,0 +1,2 @@
+nodejs 16.1.0
```

Comment:
No they don't. I think that's kind of the point of the versioning file—to have everyone on the same one. 😅 I updated it to 16.13.0!

Reply:
@meg-gutshall to be fair, most versioning systems allow you to specify to the level you care about - so `nvm` lets you specify major/minor/patch versions, just like `npm` and `yarn` - basically saying "we just need the features in version 16" or "we need the features in versions greater than 16.6" or "we need exactly the version 16.13.0" - so modern versioning files allow for this sort of thing. we'll have to update this file every time node releases a new version which is what I was hoping to avoid. I'm ok with including it and dealing with the overhead since it seems it'll be helpful, just thought I'd provide the context here.

Reply:
Ah okay. Thanks for that Dan! 😃